### PR TITLE
⭐️ add configuration to enable Linux/macOS service

### PR DESCRIPTION
**Pull request by atomic111**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -45,21 +45,28 @@ MONDOO_BINARY="cnspec" # binary that we search for
...
+MONDOO_SERVICE=''
 MONDOO_REGISTRATION_TOKEN=''
 
 print_usage() {
   echo "usage: [-i]" >&2
   echo "  Options: " >&2
   echo "    -i <installer>:  Select a specific installer, options are:" >&2
   echo "                     macOS: brew, pkg" >&2
+  echo "    -s <service>:    Enables the cnspec service for the system." >&2
+  echo "                     options are: enable" >&2
```
Comment (scottford-io) [inline install.sh:57]: @atomic111 is there an option for `disable` as well? if not, why do we have options? Why not just `-s` enables cnspec to run as a service?

File: install.sh [hunk 2]
```diff
@@ -45,21 +45,28 @@ MONDOO_BINARY="cnspec" # binary that we search for
...
   echo "    -t <token>:      Registration Token to authenticate with" >&2
   echo "                     Mondoo Platform" >&2
+  echo "    -u <updater>:    Enables the Mondoo auto updater for the system." >&2
+  echo "                     options are: enable" >&2
```
Comment (scottford-io) [inline install.sh:61]: Can this be used without `-s` or is it in addition to running cnspec as a service?

**Review by scottford-io — COMMENTED** [review]
@atomic111 a couple of suggestions for you.

File: install.sh (hunk 1 above)
Comment (atomic111) [reply install.sh:57]: i want to create the disable option later

File: install.sh (hunk 2 above)
Comment (atomic111) [reply install.sh:61]: -s and -u can be used independently of each other

**Review by chris-rock — APPROVED** [review]
Thank you @atomic111

**Pull request merged**